index.html

```diff
@@ -11,105 +11,105 @@
 </head>
 <body>
   <header>
-    <nav class="desktop-menu">
-      <div>
-        <ul>
-          <li><a href="">F</a></li>
-          <li><a href="">Twi</a></li>
-          <li><a href="">English</a></li>
-          <li><a href="">My Page</a></li>
-          <li><a href="">Logout</a></li>         
-        </ul>        
-      </div>
-      <div>
-        <div>SkillHub Academy<br>Ongoing Program</div>
-        <ul>
-          <li><a href="about.html">About</a></li>
-          <li><a href="">Courses</a></li>
-          <li><a href="">Register</a></li>
-          <li><a href="">Sponsors</a></li>
-          <li><a href="">News</a></li>
-          <li><a href="">C Academy</a></li>         
+    <nav class="desktop-menu">      
+        <ul class="icon-container dark-background display-flex">
+          <li class="icon-item"><a href=""><img src="./images/facebook.png" alt="facebook-logo"></a></li>
+          <li class="icon-item"><a href=""><img src="./images/twitter.png" alt="twitter-logo"></a></li>
+          <li class="icon-item"><a class=" white-color" href="">English</a></li>
+          <li class="icon-item"><a class=" white-color" href="">My Page</a></li>
+          <li class="icon-item"><a class=" white-color" href="">Logout</a></li>         
+        </ul>  
+      <div class="icon-container-2 display-flex">
+        <div class="display-flex">SkillHub Academy<br>Ongoing Program</div>
+        <ul class="display-flex">
+          <li><a class="icon-item red-color" href="about.html">About</a></li>
+          <li><a class="icon-item dark-color" href="#courses">Courses</a></li>
+          <li><a class="icon-item dark-color" href="#">Register</a></li>          
+          <li><a class="name-border red-color" href="">Academy</a></li>         
         </ul>         
-      </div>
-    </nav>
-    <div class="mobile-header mobile-background">  
-    <img src="./images/hamburger.png" alt="Hamburger">
-    <h1 class="pad red-color"><span class="welcome">"Welcome to SkillHub Academy!"</span><br><span class="header-ongoing">Ongoing Live<br>Skill Tutorial</span></h1>
+      </div>     
+    </nav>  
+    <div class="desktop-design mobile-header mobile-background">   
+    <div class="hamburger">    
```

Comment:
- [ ] [OPTIONAL] The mobile menu background should have the same color/pattern as the top section background or transparent background, and the menu should also have a close button

![Screenshot_20220526_193116](https://user-images.githubusercontent.com/25227022/170553704-c2b92c45-eb3f-49dc-b9b4-a7364301b481.png)

Reply:
Thanks 
Done